quilt/tools/command.py

```diff
@@ -62,6 +62,12 @@ def _save_auth(auth):
 def _handle_response(resp, **kwargs):
     if resp.status_code == requests.codes.unauthorized:
         raise CommandException("Authentication failed. Run `quilt login` again.")
+    elif not resp.ok:
```

Comment:
It's always been 200; were you thinking about the "no content" codes from nginx?
Regardless, though, `resp.ok` will be true for any 2xx status code.